lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -680,9 +681,10 @@ do_start(#server_hello{cipher_suite = SelectedCipherSuite,
         %% of the triggering HelloRetryRequest.
         ClientKeyShare = ssl_cipher:generate_client_shares([SelectedGroup]),
         TicketData = get_ticket_data(self(), SessionTickets, UseTicket),
+        OcspNonce = public_key:ocsp_nonce(OcspNonceOpt),
```

Comment:
We should only calculate nonce if stapling is enabled.

Reply:
> We should only calculate nonce if stapling is enabled and negotiated.

done